src/features/user/DonationReceipt/DonationReceiptLayout.tsx

```diff
@@ -63,6 +76,7 @@ const DonationReceiptLayout = () => {
                 dtn,
                 year,
                 challenge,
+                donorEmail: data.donor.email,
               })
             );
         }
```

Comment:
When are we removing `receiptData` from sessionStorage?

Reply:
After confirming the receipt data.